addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -51,12 +54,32 @@
 // Style
 // ------------------------------------------------------------------
 
-.o_ChatterTopbar .o-active {
-    color: $o-brand-odoo;
-    background-color: lighten(gray('300'), 7%);
-    border-color: gray('300');
-    border-bottom-color: transparent;
-    border-top-color: transparent;
+.o_ChatterTopbar_actions.o-has-active-button {
+    border-color: $border-color;
+}
+
+.o_ChatterTopbar_button {
+    border-radius: 0;
+
+    &:hover{
+        background-color: gray('300');
+    }
+
+    &.o-active {
+        color: $o-brand-odoo;
+        background-color: lighten(gray('300'), 7%);
+        border-left-color: $border-color;
```

Comment:
This is the rule I'm referring to in enterprise PR. It should not be set if the button is the first button and o-bordered is not set.